src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverImpl.java

```diff
@@ -96,6 +98,8 @@
     private final ResourceResolverContext context;
 
     protected final Map<ResourceTypeInformation,Boolean> resourceTypeLookupCache = new ConcurrentHashMap<>();
+    
+    protected Map<String,Object> propertyMap;
```

Comment:
Any particular reason to not make this private?

Reply:
fixed